packages/application-components/src/components/modal-page/internals/modal-page-top-bar.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text,
+  CloseIcon,
+  Spacings,
+  FlatButton,
+  AngleLeftIcon,
+  SecondaryIconButton,
+} from '@commercetools-frontend/ui-kit';
+import { getTopBarStyles } from './modal-page.styles';
+
+const ModalPageTopBar = props => (
+  <div css={getTopBarStyles(props)}>
+    <Spacings.Inline>
+      <FlatButton
+        tone="primary"
+        label={props.previousPath || ''}
```

Comment:
defaultProps instead?

Reply:
Ah right, and I'm actually setting it as _requiredIf showTopBar = true_

Reply:
Talked with design and this now defaults to "Go Back" (translated)